pysim/compositesystem.pyx

```diff
@@ -102,3 +145,45 @@ cdef class CompositeSystem(SimulatableSystem):
         bs_portname =  bytes(portname,'utf-8')
         bs_output_name = bytes(output_name,'utf-8')
         self._c_sys.connect_port_out(bs_portname, output_system._c_s, bs_output_name)
+        
+    def expand_single_output(self, portname, sys, sysport, initialvalue):
+        '''Method for expanding single output port'''
+        if type(initialvalue)==list:
+            self.add_port_out_vector(portname, initialvalue, '')
+            self.connect_port_out(portname, sys, sysport)
+        elif type(initialvalue)==int or type(initialvalue)==float:
+            self.add_port_out_scalar(portname, initialvalue, '')
+            self.connect_port_out(portname, sys, sysport)
+        else:
+            warning_text = 'Couldnt expand output port with portname {}'.format(portname)
+            raise ConnectionAbortedError(warning_text)
+        
+    #Help functions for setting up a composite system
+    def expand_single_port(self, name, system, initialvalue):
```

Comment:
I dont think this should be here. It is reduntant, and if there is an input with the same name as an output the result might not be what the user expects.